code/lib/cli/src/repro-templates.ts

```diff
@@ -149,6 +149,29 @@ const htmlWebpackTemplates = {
   },
 };
 
+const htmlViteTemplates = {
+  'html-vite/default-js': {
+    name: 'HTML Vite JS',
+    script: 'yarn create vite . --template vanilla && echo "export default {}" > vite.config.js',
+    cadence: ['daily', 'weekly'],
+    expected: {
+      framework: '@storybook/html-vite',
+      renderer: '@storybook/html',
+      builder: '@storybook/builder-vite',
+    },
+  },
+  'html-vite/default-ts': {
+    name: 'HTML Vite TS',
+    script: 'yarn create vite . --template vanilla-ts && echo "export default {}" > vite.config.js',
+    cadence: ['daily', 'weekly'],
```

Comment:
Maybe we should include this one in CI?  @shilman WDYT?  I see that html-webpack isn't, but maybe its good to have one of the html frameworks run in CI?

Reply:
I've updated `html-vite/default-ts` to run in `ci` too, seems like a good idea to me.

Reply:
@IanVS I've updated to add `ci` to the cadence, let me know if there any other changes you'd like.